test/integration/02-user-wallet/02-index.spec.ts

```diff
@@ -81,7 +81,7 @@ describe("UserWallet", () => {
         setUsername({ username: "3basd", id: accountId0 }),
       ).resolves.toBeInstanceOf(ValidationError)
       await expect(
-        setUsername({ username: "bc1ba", id: accountId0 }),
+        setUsername({ username: "bc1be", id: accountId0 }),
```

Comment:
problem here too

Reply:
This isn't a problem - there is no dependency on the `"bc1ba"` username.

Reply:
is a problem.. should not change param values

Reply:
not a problem on this one I think. because `bc1be` is also an invalid username type, so the test result should be the same.